lib/blueshift/index.js

```diff
@@ -89,5 +94,42 @@ Blueshift.prototype.group = function(group){
 Blueshift.prototype.track = function(track){
   var properties = track.properties();
   properties._bsft_source = 'segment.com';
+  properties.customer_id = track.userId();
+  properties.anonymousId = track.anonymousId();
+  removeBlanks(properties)
   window.blueshift.track(track.event(), properties);
 };
+
+/**
+ * Alias.
+ *
+ * @param {Alias} alias
+ */
+
+Blueshift.prototype.alias = function(alias){
+  var properties = {
+    _bsft_source: 'segment.com',
+    customer_id: alias.userId(),
+    previous_customer_id: alias.previousId(),
+    anonymousId: alias.anonymousId()
+  }
+  removeBlanks(properties);
+  window.blueshift.track('alias',properties);
+};
+
+/**
+ * Remove blank attributes from object
+ *
+ * @param {Object} obj
+ * @return {Object}
+ * @api private
+ */
+
+function removeBlanks(obj){
+  for (var i in obj) {
```

Comment:
This needs to either be wrapped in a `hasOwnProperty` check or, better, replace it with `each`